monitoring/values-prom-operator.yaml

```diff
@@ -285,4 +279,10 @@ grafana:
       - kubernetes.io/pvc-protection
     # subPath: ""
     # existingClaim:
-
+  extraVolumes:
```

Comment:
Isn't the creation of the ConfigMap optional?  If $INCLUDED_ALERTS_DIR is not set, the configMap is not created.  If the configMap is not created, won't specifying it as an extra Volume cause the pod to fail to start?  Easiest fix might be to always create the configMap even if it is an "empty" file.  Or, only add the extra volume when necessary.

Reply:
It is not optional, it will be created based on the provided ones we have. If they add more, those will just be added to the existing configmap but the configmap will always exist.